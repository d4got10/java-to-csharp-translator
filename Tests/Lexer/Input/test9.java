JPanel descriptionPanel = new JPanel();
descriptionPanel.setLayout(new GridBagLayout());
getContentPane().add(
  descriptionPanel,
  new GridBagConstraints(0, 0, 2, 1, 1.0, 0.0, GridBagConstraints.CENTER, GridBagConstraints.BOTH, new Insets(0, 0, 0,
    0), 0, 0));
descriptionPanel.setBackground(Color.white);
descriptionPanel.setBorder(BorderFactory.createMatteBorder(0, 0, 1, 0, Color.black));
  descriptionPanel.add(descriptionText, new GridBagConstraints(0, 0, 1, 1, 1.0, 0.0, GridBagConstraints.CENTER,
    GridBagConstraints.BOTH, new Insets(5, 5, 5, 5), 0, 0));
  descriptionText.setWrapStyleWord(true);
JPanel panel = new JPanel();
getContentPane().add(
  panel,
  new GridBagConstraints(0, 1, 1, 1, 1.0, 1.0, GridBagConstraints.CENTER, GridBagConstraints.NONE, new Insets(5, 5, 0,
    5), 0, 0));
panel.add(new JLabel(name + ":"));
panel.add(component);
JPanel buttonPanel = new JPanel();
getContentPane().add(
  buttonPanel,
  new GridBagConstraints(0, 2, 2, 1, 0.0, 0.0, GridBagConstraints.EAST, GridBagConstraints.NONE,
    new Insets(0, 0, 0, 0), 0, 0));
  JButton okButton = new JButton("OK");
  buttonPanel.add(okButton);
