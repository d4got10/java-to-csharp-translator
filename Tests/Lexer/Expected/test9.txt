Identifier JPanel 1:1
Identifier descriptionPanel 1:8
Operator = 1:25
Keyword new 1:27
Identifier JPanel 1:31
Operator ( 1:37
Operator ) 1:38
Semicolon ; 1:39
Identifier descriptionPanel 2:1
Operator . 2:2
Identifier setLayout 2:11
Operator ( 2:27
Keyword new 2:28
Identifier GridBagLayout 2:32
Operator ( 2:45
Operator ) 2:46
Operator ) 2:47
Semicolon ; 2:48
Identifier getContentPane 3:1
Operator ( 3:15
Operator ) 3:16
Operator . 3:18
Identifier add 3:21
Operator ( 3:21
Identifier descriptionPanel 4:3
Comma , 4:19
Keyword new 5:3
Identifier GridBagConstraints 5:7
Operator ( 5:25
Value 0 5:26
Comma , 5:27
Value 0 5:29
Comma , 5:30
Value 2 5:32
Comma , 5:33
Value 1 5:35
Comma , 5:36
Value 1.0 5:38
Comma , 5:41
Value 0.0 5:43
Comma , 5:46
Identifier GridBagConstraints 5:48
Operator . 5:49
Identifier CENTER 5:55
Comma , 5:73
Identifier GridBagConstraints 5:75
Operator . 5:76
Identifier BOTH 5:80
Comma , 5:98
Keyword new 5:100
Identifier Insets 5:104
Operator ( 5:110
Value 0 5:111
Comma , 5:112
Value 0 5:114
Comma , 5:115
Value 0 5:117
Comma , 5:118
Value 0 6:5
Operator ) 6:6
Comma , 6:7
Value 0 6:9
Comma , 6:10
Value 0 6:12
Operator ) 6:13
Operator ) 6:14
Semicolon ; 6:15
Identifier descriptionPanel 7:1
Operator . 7:2
Identifier setBackground 7:15
Operator ( 7:31
Identifier Color 7:32
Operator . 7:33
Identifier white 7:38
Operator ) 7:43
Semicolon ; 7:44
Identifier descriptionPanel 8:1
Operator . 8:2
Identifier setBorder 8:11
Operator ( 8:27
Identifier BorderFactory 8:28
Operator . 8:29
Identifier createMatteBorder 8:46
Operator ( 8:59
Value 0 8:60
Comma , 8:61
Value 0 8:63
Comma , 8:64
Value 1 8:66
Comma , 8:67
Value 0 8:69
Comma , 8:70
Identifier Color 8:72
Operator . 8:73
Identifier black 8:78
Operator ) 8:83
Operator ) 8:84
Semicolon ; 8:85
Identifier descriptionPanel 9:3
Operator . 9:4
Identifier add 9:7
Operator ( 9:23
Identifier descriptionText 9:24
Comma , 9:39
Keyword new 9:41
Identifier GridBagConstraints 9:45
Operator ( 9:63
Value 0 9:64
Comma , 9:65
Value 0 9:67
Comma , 9:68
Value 1 9:70
Comma , 9:71
Value 1 9:73
Comma , 9:74
Value 1.0 9:76
Comma , 9:79
Value 0.0 9:81
Comma , 9:84
Identifier GridBagConstraints 9:86
Operator . 9:87
Identifier CENTER 9:93
Comma , 9:111
Identifier GridBagConstraints 10:5
Operator . 10:6
Identifier BOTH 10:10
Comma , 10:28
Keyword new 10:30
Identifier Insets 10:34
Operator ( 10:40
Value 5 10:41
Comma , 10:42
Value 5 10:44
Comma , 10:45
Value 5 10:47
Comma , 10:48
Value 5 10:50
Operator ) 10:51
Comma , 10:52
Value 0 10:54
Comma , 10:55
Value 0 10:57
Operator ) 10:58
Operator ) 10:59
Semicolon ; 10:60
Identifier descriptionText 11:3
Operator . 11:4
Identifier setWrapStyleWord 11:20
Operator ( 11:35
Value true 11:36
Operator ) 11:40
Semicolon ; 11:41
Identifier JPanel 12:1
Identifier panel 12:8
Operator = 12:14
Keyword new 12:16
Identifier JPanel 12:20
Operator ( 12:26
Operator ) 12:27
Semicolon ; 12:28
Identifier getContentPane 13:1
Operator ( 13:15
Operator ) 13:16
Operator . 13:18
Identifier add 13:21
Operator ( 13:21
Identifier panel 14:3
Comma , 14:8
Keyword new 15:3
Identifier GridBagConstraints 15:7
Operator ( 15:25
Value 0 15:26
Comma , 15:27
Value 1 15:29
Comma , 15:30
Value 1 15:32
Comma , 15:33
Value 1 15:35
Comma , 15:36
Value 1.0 15:38
Comma , 15:41
Value 1.0 15:43
Comma , 15:46
Identifier GridBagConstraints 15:48
Operator . 15:49
Identifier CENTER 15:55
Comma , 15:73
Identifier GridBagConstraints 15:75
Operator . 15:76
Identifier NONE 15:80
Comma , 15:98
Keyword new 15:100
Identifier Insets 15:104
Operator ( 15:110
Value 5 15:111
Comma , 15:112
Value 5 15:114
Comma , 15:115
Value 0 15:117
Comma , 15:118
Value 5 16:5
Operator ) 16:6
Comma , 16:7
Value 0 16:9
Comma , 16:10
Value 0 16:12
Operator ) 16:13
Operator ) 16:14
Semicolon ; 16:15
Identifier panel 17:1
Operator . 17:2
Identifier add 17:5
Operator ( 17:10
Keyword new 17:11
Identifier JLabel 17:15
Operator ( 17:21
Identifier name 17:22
Operator + 17:27
Value ":" 17:29
Operator ) 17:30
Operator ) 17:31
Semicolon ; 17:32
Identifier panel 18:1
Operator . 18:2
Identifier add 18:5
Operator ( 18:10
Identifier component 18:11
Operator ) 18:20
Semicolon ; 18:21
Identifier JPanel 19:1
Identifier buttonPanel 19:8
Operator = 19:20
Keyword new 19:22
Identifier JPanel 19:26
Operator ( 19:32
Operator ) 19:33
Semicolon ; 19:34
Identifier getContentPane 20:1
Operator ( 20:15
Operator ) 20:16
Operator . 20:18
Identifier add 20:21
Operator ( 20:21
Identifier buttonPanel 21:3
Comma , 21:14
Keyword new 22:3
Identifier GridBagConstraints 22:7
Operator ( 22:25
Value 0 22:26
Comma , 22:27
Value 2 22:29
Comma , 22:30
Value 2 22:32
Comma , 22:33
Value 1 22:35
Comma , 22:36
Value 0.0 22:38
Comma , 22:41
Value 0.0 22:43
Comma , 22:46
Identifier GridBagConstraints 22:48
Operator . 22:49
Identifier EAST 22:53
Comma , 22:71
Identifier GridBagConstraints 22:73
Operator . 22:74
Identifier NONE 22:78
Comma , 22:96
Keyword new 23:5
Identifier Insets 23:9
Operator ( 23:15
Value 0 23:16
Comma , 23:17
Value 0 23:19
Comma , 23:20
Value 0 23:22
Comma , 23:23
Value 0 23:25
Operator ) 23:26
Comma , 23:27
Value 0 23:29
Comma , 23:30
Value 0 23:32
Operator ) 23:33
Operator ) 23:34
Semicolon ; 23:35
Identifier JButton 24:3
Identifier okButton 24:11
Operator = 24:20
Keyword new 24:22
Identifier JButton 24:26
Operator ( 24:33
Value "OK" 24:34
Operator ) 24:35
Semicolon ; 24:36
Identifier buttonPanel 25:3
Operator . 25:4
Identifier add 25:7
Operator ( 25:18
Identifier okButton 25:19
Operator ) 25:27
Semicolon ; 25:28
